### ISSUE TITLE
Scrollable list-box example doesn’t work on iOS voiceover in Safari

### ISSUE THREAD

**Issue by jknanda78** [w3c/aria-practices#631]
Hello,

I tried list-box srollable example (https://www.w3.org/TR/wai-aria-practices/examples/listbox/listbox-scrollable.html) on **iOS** **Safari** with **Voiceover** and found that it doesn’t work as expected.

 Issue: The focus is trapped on the first/selected item in the list. I can go back but cannot go forward, meaning to the next option(s) in the list.

Not sure if anyone has reported this issue. Please let me know in case any flaw in my testing.

Thanks in advance

**Comment by jnurthen** [comment]
Confirmed. I can reproduce. Thanks for the issue report.

**Comment by jknanda78** [comment]
Thank you for the confirmation. Do you have any ETA? Actually I am building a component and planning to start with A11Y-Mobile first design implementation.

Thanks again.

**Comment by jnurthen** [comment]
No. If you find the issue we would welcome a PR.

**Comment by abhipanda** [comment]
Is there any workaround ?
